realue/worker/pdusessworker/handler.go

```diff
@@ -26,6 +60,188 @@ const (
 	IPV4_MIN_HEADER_LEN int = 20
 )
 
+func checksum(buf []byte) uint16 {
```

Comment:
There is similar function in the util package, should we reuse it ?

Reply:
done